zksync/sdk/musig/schnorr_musig_signer.py

```diff
@@ -0,0 +1,89 @@
+from typing import List
+
+from zksync.sdk.musig.schnorr_musig_error import SchnorrMusigError
+from zksync.sdk.musig.schnorr_musig_native import *
+
+
+class SchnorrMusigSigner:
+
+    def __init__(self, musig: SchnorrMusigNative, signer: MusigSignerPointer, public_key: List[int]) -> None:
+        self.musig = musig
+        self.signer = signer
+        self.public_key = public_key
+
```

Comment:
The `public_key` field/argument is misleading since you pass an encoded list of public keys, not a single one.

Reply:
Actually, it is a single public_key.

I am trying to keep python sdk similar to java sdk. I have talked with @MaximFischuk and he had a following vision:
SchnorrMusigSigner represents signer of a specific user and it stores only a single key.
On the other side, we may need AggSchnorrMusigSigner in the future. As a result, it will contain agg_public_keys and previously mentioned methods `verify`.

I did not have intention to create AggSchnorrMusigSigner right now, cause do not want to blow the scope of my first task. It is just initial commit with minimal list of wrappers and C lib bindings.

We can create AggSchnorrMusigSigner wrapper in the next PR.